README.md

```diff
@@ -333,6 +334,8 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Finance
 
+* [**Aaru**](https://github.com/KSSidll/Arru) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/com.kssidll.arru)]**</sup>
+* [**FDTracker**](https://github.com/Abhay-cloud/Fixed-Deposit-Tracker) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/dev.abhaycloud.fdtracker)]**</sup>
```

Comment:
The 'official' name seems to be *Fixed Deposit Tracker*.

```suggestion
* [**Fixed Deposit Tracker**](https://github.com/Abhay-cloud/Fixed-Deposit-Tracker) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/dev.abhaycloud.fdtracker)]**</sup>
```

Reply:
https://github.com/Abhay-cloud/Fixed-Deposit-Tracker#home-screen
They use short name "FDTracker" when installed.

Reply:
I see.A little confusing, but hat's just how it is. :smile: